styles/layout.css

```diff
@@ -1,14 +1,27 @@
-.template {
-  display: none;
+#articles {
   text-align: center;
-  margin: 0 auto;
 }
 
-a {
```

Comment:
for me the css is easier to read if you group the parent/children elements together.  for example, you got an a tag getting styled in between the #article and #article img.